lib/graphql/schema/build_from_definition.rb

```diff
@@ -22,18 +22,60 @@ def self.call(type, field, obj, args, ctx)
 
       # @api private
       class ResolveMap
+        
         def initialize(resolve_hash)
-          @resolve_hash = resolve_hash
+          @resolve_hash = convert_keys_to_strings(resolve_hash)
+        end
+
+        def convert_keys_to_strings(h)
```

Comment:
Can some of these new methods be hidden under a `private` declaration? That will make it a bit easier to maintain, since the reader knows those methods are only called within the class itself.